core/cli/main.py

```diff
@@ -112,6 +153,9 @@ async def async_main(
     if not ui_started:
         return False
 
+    if not await llm_api_check(ui):
```

Comment:
I previously initialized `telemetry.start()` and then did `telemetry.send()` in `run_project()`. Because it is now needed before that, I think it makes sense to do `start()` before this line, but we don't want to run it for list/config/import/... operations, I think it makes sense to split this function into two:

1. async_main() that runs everything before this line, and then runs the second one
2. run_pythagora_session() that runs everything from this line down

Then the async main can have telemetry start, run the new function, telemetry send.

(and the start and send can be removed from run_project). This will minimize duplication and still make it start/end correctly

Lemme know if that makes sense